### PR TITLE
fixed no test assemblies found issue

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
fixed no test assemblies found issue

### CODE REVIEW

File: Tasks/VsTestV2/distributedtest.ts [hunk 1]
```diff
@@ -36,15 +36,12 @@ export class DistributedTest {
             }
         } catch (error) {
             ci.publishEvent({ environmenturi: this.inputDataContract.RunIdentifier, error: error });
-            tl.error(error);
```
Comment (vagisha-nidhi) [inline Tasks/VsTestV2/distributedtest.ts]: Why have we removed this?

Comment (ShreyasRmsft) [reply Tasks/VsTestV2/distributedtest.ts]: tl.setResult logs the error

**Comment by vincentn365** [comment]
Hi, 
Is this supposed to fix the error below from Execute Tests task which I'm getting for the past 3 days?
##[error]Error: Preparing the test sources file failed. Error : Error: No test sources found matching the given filter '**\*Test*.dll'
##[error]VsTest task failed.

**Comment by ShreyasRmsft** [comment]
@vincent1173 along with https://github.com/microsoft/azure-pipelines-tasks/pull/11919 will fix this. We are working on hotfixing this at the soonest possible

**Pull request merged**